apps/api/src/app/shared/services/in-memory-provider/elasticache-cluster-provider.ts

```diff
@@ -84,15 +85,17 @@ export const getElasticacheCluster = (): Cluster | undefined => {
     enableReadyCheck: true,
     redisOptions: {
       tls: {},
+      connectTimeout: 10000,
     },
     // scaleReads: 'slave' as NodeRole, // Enable it for improved performance to read only in replicas
     /*
      *  Disabled in Prod as affects performance
      */
     showFriendlyErrorStack: process.env.NODE_ENV !== 'prod',
-    slotsRefreshTimeout: 5000,
+    slotsRefreshTimeout: 10000,
   };
 
+  Logger.log(`Initializing Elasticache Cluster with ${instances?.length} instances`);
```

Comment:
This is not going to count the real instances as the set up for Elasticache is different. Not sure if through the URL or any API call we would know the exact number.